jenkins/job-configs/kubernetes-jenkins/bootstrap-ci.yaml

```diff
@@ -45,3 +45,23 @@
         branch: master
         job-name: ci-kubernetes-e2e-gce-flaky
         repo-name: k8s.io/kubernetes
+    - kubernetes-e2e-gke-autoscaling:
+        branch: master
+        job-name: ci-kubernetes-e2e-gke-autoscaling
+        repo-name: k8s.io/kubernetes
```

Comment:
And I feel like we can move `branch` and `repo-name` to top, and generate `job-name` from each entry since they are all the same..